### DISCUSSION TITLE
Add support of Knooppuntnet

### DISCUSSION THREAD

**Discussion by yves-jaeger** [organicmaps/organicmaps#7563] [Ideas]
Knooppuntnet contains walking and bicycle node networks.
It would be very nice, if organicmaps shows the node network (either for hiking of biking) of the current area (e.g. as overlay?).

For more details of Knooppuntnet see:
* https://wiki.openstreetmap.org/wiki/Knooppuntnet
* https://knooppuntnet.nl/

**Comment by biodranik** [comment]
Is it related to https://github.com/organicmaps/organicmaps/issues/2185 ?

**Comment by d33psky** [comment]
The #2185 is certainly related. knooppuntnet is probably a generic extension of it.

The [https://knooppuntnet.nl/](url) website and apps like 'fietsknoop' [https://play.google.com/store/apps/details?id=nl.vv.fietsknoop](url) all relate to physical direction indicators / nodes throughout several countries. I use the biking and hiking ones the most.

An overlay with numbered nodes and routes where one would tap nodes in order to build a track would be ideal. When biking or hiking the track these numbers with directions for the next number allow for easy path following. Like 'you are arriving at node 53, the next node is 21' and on the physical direction indicators these numbers with all other nearest numbers with arrows are listed.

I've tried to mimic the knooppuntnet often by setting a 'route to' next to my begin point, and then 'add stop' on all major junctions to create a track but organicmaps' routing system always gets confused as it tries to make shorter routes or wants to change the node order. And of course I do not know the locations of all direction indicators in the field so I often miss the best scenery.
